src/select.scss

```diff
@@ -87,6 +88,10 @@ $block: #{$fd-namespace}-select;
     font-size: var(--sapFontSize);
   }
 
+  &__item[aria-selected=true] {
+    border-bottom: 0 !important;
+  }
+
```

Comment:
It's not fiori3 compatible. First pictures on specifications are wrong.
![image](https://user-images.githubusercontent.com/26483208/83498993-ede16500-a4bc-11ea-99ad-8785097d633a.png)
Anyway, we don't use important at all. If this situation occurs, you would need to use stronger selector

Reply:
So you we don't need to have `fd-select__item` on every single item anymore.

Reply:
Which spec do we want to follow as the source of truth? https://experience.sap.com/fiori-design-web/select/
![Screen Shot 2020-06-02 at 8 05 40 AM](https://user-images.githubusercontent.com/29607818/83536476-1f9c0100-a4a8-11ea-9fed-152bc2997ece.png)

Reply:
I can only assume, that we should follow list component there. Same as with multi input or combobox. Let's ask designers maybe, because spec shows 2 different views.